en/controllers/components/authentication.rst

```diff
@@ -289,9 +289,14 @@ by including them in AuthComponents authenticate array::
 
     $this->Auth->config('authenticate', [
         'Openid', // app authentication object.
-        'AuthBag.Combo', // plugin authentication object.
+        'AuthBag.Openid', // plugin authentication object.
     ]);
 
+.. note::
+    Note that when using simple notation there's no 'Authenticate' word when
+    initiating the authentication object. Instead, if using namespaces, you'll need
+    to set the full namespace of the class (including the 'Authenticate' word).
```

Comment:
This is actually how class names are specified throughout the framework. It should be on general conventions page rather than as a note here.